FrameworkSystem/Service/SecurityLoggingHandler.py

```diff
@@ -29,7 +29,7 @@ def initializeSecurityLoggingHandler(serviceInfo):
 
   try:
     testFile = "%s/seclog.jarl.test" % dataPath
-    fd = file(testFile, "w")
+    fd = open(testFile, "w")
```

Comment:
```suggestion
    with open(testFile, "w"):
      pass
```
and delete the next line